### DISCUSSION TITLE
Adding cost to the datasets tables

### DISCUSSION THREAD

**Discussion by ladislasdellinger** [langfuse/langfuse#1331] [Ideas]
### Describe the feature or potential improvement

It would be nice to add the cost of each run item (and then each run) of a dataset to monitor the cost changes between different experiments !

**Comment by marcklingen** [comment]
That's on the short term roadmap! Open to support any contributions on this as well

**Reply by ladislasdellinger** [reply]
Hi @marcklingen, here is the [PR](https://github.com/langfuse/langfuse/pull/1344) !

I'm open to any feedback on it 🙂
